scene/gui/code_edit.cpp

```diff
@@ -2235,6 +2235,10 @@ void CodeEdit::request_code_completion(bool p_force) {
 }
 
 void CodeEdit::add_code_completion_option(CodeCompletionKind p_type, const String &p_display_text, const String &p_insert_text, const Color &p_text_color, const Ref<Resource> &p_icon, const Variant &p_value, int p_location) {
+	_add_code_completion_option(p_type, p_display_text, p_insert_text, p_text_color, p_icon, p_value, p_location, {});
+}
+
+void CodeEdit::_add_code_completion_option(CodeCompletionKind p_type, const String &p_display_text, const String &p_insert_text, const Color &p_text_color, const Ref<Resource> &p_icon, const Variant &p_value, int p_location, const Vector<ScriptLanguage::TextEdit> &additional_edits) {
```

Comment:
This happened because I didn't know how to properly get `TextEdit` and `CodePos` exposed to scripting, and `add_code_completion_option` is available to scripting. The ClassDB bind methods are unable to handle the extra default argument, so the original `add_code_completion_option` became a bouncer to this method.